src/main/groovy/com/bmuschko/gradle/docker/internal/RegistryAuthLocator.groovy

```diff
@@ -249,6 +249,7 @@ class RegistryAuthLocator {
         String credentialHelperName = commandPathPrefix + credHelper
 
         String data = runCommand(hostName, credentialHelperName)
+        logger.debug('Credential helper response: {}', data)
```

Comment:
I am sure this will be helpful in identifying the underlying issues reported by users.